### PR TITLE
Etrip api endpoints

### PR DESCRIPTION
**Pull request by robertavram**

### CODE REVIEW

File: EquiTrack/users/views.py [hunk 1]
```diff
@@ -48,6 +55,12 @@ def get_object(self, queryset=None, **kwargs):
         return user
 
 
+class UsersView(ListAPIView):
```
Comment (ntrncic) [inline EquiTrack/users/views.py]: what's this view used for?

Comment (robertavram) [inline EquiTrack/users/views.py]: retrieving all users in a country. just names and ids... for the next feature that will allow a programme officer to assign action points from the app.

Comment (ntrncic) [inline EquiTrack/users/views.py]: ok cool sounds good.

File: EquiTrack/trips/models.py [hunk 2]
```diff
@@ -271,13 +271,21 @@ def save(self, **kwargs):
 
         super(Trip, self).save(**kwargs)
 
+    @property
+    def all_files(self):
+        return FileAttachment.objects.filter(object_id=self.id)
+
+    @property
+    def all_aps(self):
```
Comment (jamescw) [inline EquiTrack/trips/models.py]: change to more verbose name like `all_actionpoints` less guess work if reading from another file

File: EquiTrack/trips/urls.py [hunk 3]
```diff
@@ -2,7 +2,10 @@
 
 from django.conf.urls import patterns, url
 
-from .views import TripsApprovedView, TripsByOfficeView, TripsDashboard, TripsListApi, TripActionView, TripDetailsView
+from .views import TripsApprovedView, \
+    TripsByOfficeView, TripsDashboard, \
+    TripsListApi, TripActionView, \
+    TripDetailsView
```
Comment (jamescw) [inline EquiTrack/trips/urls.py]: wrap in brackets instead

**Pull request merged**